core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -237,6 +241,10 @@ object ConsoleProducer {
             .withRequiredArg
             .describedAs("producer_prop")
             .ofType(classOf[String])
+    val producerConfigOpt = parser.accepts("producer.config", "Producer config properties file.")
```

Comment:
Could we add in the description that producer-property takes precedence over producer.config ?